Samples/Samples.Tizen/Program.cs

```diff
@@ -13,6 +13,7 @@ protected override void OnCreate()
             base.OnCreate();
 
             MaterialComponents.Init(DirectoryInfo.Resource);
+            Xamarin.Essentials.Platform.MainWindow = MainWindow;
```

Comment:
Maybe we should follow our current "init" pattern and do something along these lines:

```
Xamarin.Essentials.Init(MainWindow);
```

Reply:
I also see a lot of `Platform.MainWindow ?? GetStaticThing`. What is the difference in values or functionality when using either?

Reply:
I will change to Init pattern.